server/lib/paypal.ts

```diff
@@ -0,0 +1,25 @@
+import paypalPayoutsSDK from '@paypal/payouts-sdk';
+
+interface ConnectedAccount {
+  token: string;
+  clientId: string;
+}
+
+const getPayPalClient = ({ token, clientId }: ConnectedAccount) => {
+  if (process.env.NODE_ENV === 'production') {
+    return new paypalPayoutsSDK.core.LiveEnvironment(clientId, token);
+  }
+  const environment =
+    process.env.NODE_ENV === 'production'
+      ? new paypalPayoutsSDK.core.LiveEnvironment(clientId, token)
+      : new paypalPayoutsSDK.core.SandboxEnvironment(clientId, token);
```

Comment:
This check seems unnecessary given that we already check `if (process.env.NODE_ENV === 'production')` above

Reply:
Oh, a wrong partial refactor!
I actually need to return the PayPalHttpClient, this would break in production.